shared/actions/platform-specific/index.desktop.tsx

```diff
@@ -204,7 +204,7 @@ function* checkRPCOwnership(_, action: ConfigGen.DaemonHandshakePayload) {
 }
 
 const initOsNetworkStatus = (state, action) =>
-  ConfigGen.createOsNetworkStatusChanged({isInit: true, online: navigator.onLine, type: 'unknown'})
+  ConfigGen.createOsNetworkStatusChanged({isInit: true, online: navigator.onLine, type: 'wifi'})
```

Comment:
why?

Reply:
was worried someone would get the state when not on mobile and disable something since the state wasn't `wifi`

Reply:
i think the background services should treat unknown that way instead of lying about being on wifi

Reply:
yeah could wrap it in a method that allows both, just seems more error prone if someone doesn't use it properly. since we have no info for desktop i think `wifi` is a more sensible default